docs/DeveloperGuide.md

```diff
@@ -1247,29 +1247,129 @@ testers are expected to do more *exploratory* testing.
     1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+1. Shut down application
 
+    1. Prerequisites: Application must currently be running.
+    
+    2. Test case: Close the application using any means other than the command `exit`.
+       Expected: The application exits.
+
+    2. Test case: Close the application using the command `exit`.
+       Expected: The application exits.
+       
 ### Removing a task
 
 1. Removing a task while all tasks are being shown
 
     1. Prerequisites: All tasks (done and undone) are shown, and user is not currently cd-ed into a module.
 
```

Comment:
```suggestion
    1. Prerequisites: All tasks (at least one, done and undone) are shown, and user is not currently cd-ed into a module.

```
I think this should be added otherwise there's nothing to remove.